public/app/features/library-panels/components/LibraryPanelsView/actions.ts

```diff
@@ -17,6 +17,7 @@ export function searchForLibraryPanels(args: SearchArgs): DispatchResult {
         name: args.searchString,
         perPage: args.perPage,
         page: args.page,
+        excludeUid: args.currentPanelId,
```

Comment:
Fixed an unrelated issue that I found, forgot to add this when I refactored the search previously.